### ISSUE TITLE
[Regular Face loss]

### ISSUE THREAD

**Issue by Vietdung113** [KevinMusgrave/pytorch-metric-learning#1]
Did you implement [Regular face loss](http://kaizhao.net/regularface) in this project?

**Comment by KevinMusgrave** [comment]
Thanks for the suggestion! It looks like it should be pretty easy to adapt their code for this library, so I'll definitely add it to my to-do list.

**Comment by KevinMusgrave** [comment]
I've added RegularFaceRegularizer. It can be passed into any loss function that extends WeightRegularizerMixin. At the moment, this includes ArcFaceLoss, NormalizedSoftmaxLoss, and ProxyNCALoss.

Here's an example:
```python
from pytorch_metric_learning import losses, regularizers
R = regularizers.RegularFaceRegularizer()
loss = losses.ArcFaceLoss(margin=30, num_classes=100, embedding_size=128, regularizer=R)
```
